timify-ui/src/components/EpisodeDetail/EpisodeDetail.jsx

```diff
@@ -5,15 +5,18 @@ import "./EpisodeDetail.css"
 const EpisodeDetail = ({episode}) => {
 
   return (
-    <Link to={`/podcast/episode/${episode.uuid}`}>
+    <>
```

Comment:
does this need to be here?

Reply:
the link? of course! that's how the pages get linked. am i missing something?

Reply:
> We are seeing that the component is deleted and an empty <> shown here. We don't need empty angle brackets here

oh it's just empty but it could be a div itself. it's there because there has to be a set of angle brackets wrapping the entire return statement. else all the code would be underlined red